### PR TITLE
Update Definition solves #234

### PR DESCRIPTION
**Pull request by carsten-kuebler**
This PR refers to issue #234

### CODE REVIEW

**Comment by carsten-kuebler** [comment]
@LudwigFriedmannBMW I updated the documentation and checked the other definitions (acceleration and orientation rate). I add notes to sensor data and detected objects.

File: osi_common.proto [hunk 1]
```diff
@@ -346,7 +345,7 @@ message BaseMoving
 
     // The relative orientation rate of the moving object w.r.t. its parent
     // frame and parent orientation rate in the center point of the bounding box
-    // (origin of the bounding box frame).
+    // (origin of the bounding box frame), noted in the parent frame.
```
Comment (LudwigFriedmannBMW) [inline osi_common.proto]: The orientation rate becomes global/absolute if the parent frame is not rotating.

File: osi_common.proto [hunk 2]
```diff
@@ -325,18 +325,17 @@ message BaseMoving
     //
```
Comment (LudwigFriedmannBMW) [inline osi_common.proto]: Please add to line 265 & 319: The relative orientation of the stationary object w.r.t. its parent frame, noted in the parent frame. The orientation becomes global/absolute if the parent frame is inertial .

**Comment by carsten-kuebler** [comment]
@LudwigFriedmannBMW I also add (all parent frames up to ground truth)

**Pull request merged**